site/src/components/popover.md

```diff
@@ -71,3 +71,26 @@ Popover is left aligned by default, but also can be centered or right aligned re
 	<div class="popover-content border">Popover content.</div>
 </details>
 ```
+
```

Comment:
Wondering if we can omit this documentation until or re-write it to be a suggested implementation. We don't yet have a way of sharing the front end scripts with other consumers (we're documenting TS that is not technically part of our css package)

Reply:
Oh, you're right, I always forget we don't share the scripts right now. Decided to omit this portion of documentation until we do!